### PR TITLE
Correct the button name

### LINKED ISSUE #7: Correct the name of the button(s)
**Is your feature request related to a problem? Please describe.**
The instruction for **[:keyboard: Activity: Suggest changes](../blob/main/README.md/#keyboard-activity-suggest-changes)** says, "After the commment form appears, click the **Insert a suggestion** button," but I found an **Add a suggestion** button instead of an **Insert a suggestion** one. Also, the button described on the last line actually says "**Add single comment**".

**Describe the solution you'd like**
The instruction should be modified so that it can mean the correct buttons.
- **Insert a suggestion** &rarr; **Add a suggestion**
- (**Add a single comment** &rarr; **Add single comment**)

**Describe alternatives you've considered**
The **Add single comment** button should be changed into an **Add a single comment** one.

**Additional context**
The instructions for the **Activity** is as follows:

1. On the pull request, click **Files changed**.
1. Find the `index.html` changes.
1. Hover your cursor next to the line numbers on the left side of the page.
1. Click the blue plus icon.
1. After the commment form appears, click the **Insert a suggestion** button.
1. Edit the suggestion.
1. Click **Add a single comment**.

### PR DESCRIPTION
**Pull request by sinsukehlab**
### Why:

Closes #7

### What's being changed:

README.md
**Insert a suggestion** &rarr; **Add a suggestion**

### Check off the following:

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by sinsukehlab** [comment]
I have added some commits to clarify the instruction.
Cf. [discussion comment](https://github.com/orgs/skills/discussions/9#discussioncomment-3048970)

File: README.md [hunk 1]
```diff
@@ -133,10 +133,14 @@ Now that you have explored the different ways you can review a pull request it i
...
 1. Click the blue plus icon.
-1. After the commment form appears, click the **Insert a suggestion** button.
+1. After the commment form appears, click the **Add a suggestion** button.
 1. Edit the suggestion.
 1. Click **Add a single comment**.
 
+The **Add a suggestion** button is shown below.
+
+![Add a suggestion octicon-file-diff](images/octicon-file-diff.svg)
+
```
Comment (fishviche) [inline README.md:143]: No idea

File: README.md [hunk 2]
```diff
@@ -133,10 +133,14 @@ Now that you have explored the different ways you can review a pull request it i
...
 Next, wait about 20 seconds for actions to run, then refresh this course's README (the one you're following instructions from) and a [GitHub Action](https://docs.github.com/en/actions) will automatically close this step and open the next one.
```
Comment (fishviche) [inline README.md:144]: Big text

**Review by fishviche — COMMENTED** [review]
New comments

**Review by fishviche — COMMENTED** [review]
SDFSFDSDG

File: README.md [hunk 3]
```diff
@@ -133,10 +133,14 @@ Now that you have explored the different ways you can review a pull request it i
 1. Find the `index.html` changes.
 1. Hover your cursor next to the line numbers on the left side of the page.
 1. Click the blue plus icon.
-1. After the commment form appears, click the **Insert a suggestion** button.
+1. After the commment form appears, click the **Add a suggestion** button.
```
Comment (sinsukehlab) [inline README.md:136]: Fix a typo #9
```suggestion
1. After the comment form appears, click the **Add a suggestion** button.
```

**Comment by sinsukehlab** [comment]
I have added the icon of the button in my commits on July 8th.
Are there any problems with it?
> Yet it is still difficult to locate that tricky button - Can you add the icon of the "Add a suggestion" button to the README file ? Or even a small screenshot where it is found ?

**Comment by sinsukehlab** [comment]
Gentle reminder
@dihydroJenoxide I would be grateful if you could review this pull request as soon as possible.

**Review by dihydroJenoxide — APPROVED** [review]
We appreciate your interest in improving this Skills course! ✨ 
I made a few changes to match our [style guide](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#content-style-guide-for-github-docs-) before merging. 
Thank you for your contribution 💛

File: README.md [hunk 4]
```diff
@@ -133,7 +133,8 @@ Now that you have explored the different ways you can review a pull request it i
 1. Find the `index.html` changes.
 1. Hover your cursor next to the line numbers on the left side of the page.
 1. Click the blue plus icon.
-1. After the commment form appears, click the **Insert a suggestion** button.
+1. After the comment form appears, click the **Add a suggestion** button.
```
Comment (dihydroJenoxide) [inline README.md:136]:
```suggestion
1. After the comment form appears, click the **Add a suggestion** button. <br>
```

File: README.md [hunk 5]
```diff
@@ -133,10 +133,14 @@ Now that you have explored the different ways you can review a pull request it i
...
 1. Edit the suggestion.
 1. Click **Add a single comment**.
 
+The **Add a suggestion** button is shown below.
+
+![Add a suggestion octicon-file-diff](images/octicon-file-diff.svg)
+
```
[the remark this answers is not in the document]
Comment (SerhiyKhrus777777) [reply README.md]: Readme

**Pull request merged**